sdk/communication/Azure.Communication.Calling.Server/src/CallClient.cs

```diff
@@ -365,25 +365,25 @@ public virtual Response<PlayAudioResponse> PlayAudio(string callLegId, PlayAudio
             }
         }
 
-        /// <summary> Invite participants to a call. </summary>
+        /// <summary> Add a participant to the call. </summary>
         /// <param name="callLegId"> The call leg id. </param>
-        /// <param name="participants"> The uri of the audio file. </param>
-        /// <param name="alternateCallerId">The flag to indicate if audio file need to be played in a loop.</param>
+        /// <param name="participant"> The identity of participant to be added to the call. </param>
         /// <param name="operationContext">The operation context. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <param name="alternateCallerId">The phone number to use when adding a pstn participant.</param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        /// <exception cref="ArgumentNullException"> <paramref name="participants"/> is null. </exception>
-        public virtual async Task<Response> InviteParticipantsAsync(string callLegId, IEnumerable<CommunicationIdentifier> participants, string alternateCallerId = default, string operationContext = null, CancellationToken cancellationToken = default)
+        /// <exception cref="ArgumentNullException"> <paramref name="participant"/> is null. </exception>
+        public virtual async Task<Response> AddParticipantAsync(string callLegId, CommunicationIdentifier participant, string operationContext, string alternateCallerId = default, CancellationToken cancellationToken = default)
```

Comment:
I think operationContext is optional 

can we move it just before CancellationToken? for all apis.